### ISSUE TITLE
soundcloud.com

### ISSUE THREAD

**Issue by KonoromiHimaries** [yourduskquibbles/webannoyances#104]
![2018-11-22-soundcloud sluchaj bezplatnie muzyki i podcastow w serwisie soundcloud](https://user-images.githubusercontent.com/5884000/48921781-64ea7600-eea2-11e8-848a-54f63b95685c.png)

**Comment by krystian3w** [comment]
OMG, this bar is on bottom sticked.

**Comment by krystian3w** [comment]
My proposition:

```css
soundcloud.com##.m-hideHeader > #app > .m-visible.g-z-index-control-bar.playControls:style(position: absolute !important; top: 0 !important; margin-top: 0 !important; margin-bottom: 46px !important;)
soundcloud.com##.m-hideHeader > #app:style(padding-top: 46px !important;)
```

**Comment by yourduskquibbles** [comment]
I think @krystian3w proposed fix that I merged in 84428d7 is good, it moves the playbar to the top. Only other option I see is leaving the playbar stuck at the bottom so site functionality isn't "broken".
